src/js/modules/wallet/controllers/settings/profile/settings.profile.ctrl.js

```diff
@@ -0,0 +1,285 @@
+angular.module("blocktrail.wallet")
+    .controller("SettingsProfileCtrl", SettingsProfileCtrl);
+
+function SettingsProfileCtrl($scope, $rootScope, $translate, dialogService, CONFIG, $modal, formSettingsService, settingsService) {
+
+    var savedSettings = {
+        username: "",
+        email: "",
+        localCurrency: "USD",
+        language: "en",
+        receiveNewsletter: false,
+        profilePic: null
+    };
+
+    var settings = settingsService.getReadOnlySettingsData();
+
+    var listenerFormSettings;
+
+    $scope.hasEmailChanged = false;
+    $scope.securityInfo = {};
+
+    $rootScope.pageTitle = 'SETTINGS';
+
+    // this automatically updates an already open modal instead of popping a new one open
+    // TODO remove it after moving modals change password, enable/disable 2FA
+    $scope.alert = dialogService.alertSingleton();
+
+    $scope.isLoading = true;
+    $scope.isSubmitFormSettingsBtnDisabled = true;
+
+    $scope.formSettings = {
+        username: "",
+        email: "",
+        localCurrency: "USD",
+        language: "en",
+        receiveNewsletter: false,
+        profilePic: "/" + CONFIG.STATICSURL + "/img/blank_profile.png"
+    };
+
+    $scope.errors = {
+        email: false,
+        name: false
+    };
+
+    // $scope.languages = formSettingsService.getLanguages();
+    // $scope.currencies = formSettingsService.getCurrencies();
```

Comment:
uncomment languages and currencies and define them with empty array $scope.languages = [];